pil_analyzer/src/evaluator.rs

```diff
@@ -130,12 +137,27 @@ impl<'a, T: FieldElement, C: Custom> Value<'a, T, C> {
                 format!("[{}]", elements.iter().map(|e| e.type_name()).format(", "))
             }
             Value::Closure(c) => c.type_name(),
+            Value::BuiltinFunction(b) => format!("builtin{b:?}"),
```

Comment:
Actually this should usually be the type name of a closure. Let me make it proper.

Reply:
ok, turns out we don't even know the types in a closure.

What about `builtin_`? I don't particularly like `::` because it is not a namespace.

As soon as we add proper types, this will be replaced anyway.